migrations/versions/e1b5fbe3140a_add_authentication_logs.py

```diff
@@ -0,0 +1,35 @@
+"""add authentication_logs
+
+Revision ID: e1b5fbe3140a
+Revises: 691dba936e64
+Create Date: 2024-06-20 10:37:10.690565
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'e1b5fbe3140a'
+down_revision = '691dba936e64'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('authentication_logs',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('user_id', sa.Integer(), nullable=False),
+    sa.Column('successful', sa.Boolean(), nullable=False),
+    sa.Column('timestamp', sa.DateTime(), nullable=False),
+    sa.ForeignKeyConstraint(['user_id'], ['users.id'], ),
+    sa.PrimaryKeyConstraint('id')
```

Comment:
does this table need an index on user_id + timestamp + successful?

Reply:
Good idea. In https://github.com/scidsg/hushline/pull/376/commits/2fb60ede18fe68b089ac739dad263a5c0cd70cf0 I recreated the migration, and this time added an index.